### PR TITLE
[EDU-3086] feat: add secure overview

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new Secure Overview.

Note: I'll add the image ASAP.

[EDU-3086]: https://aziontech.atlassian.net/browse/EDU-3086?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/overview/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+permalink: /documentation/products/secure/overview/
+---
+
+import Button from '~/components/Button.astro'
+
+Azion Secure products assist you in protecting your domains, applications, and infrastructure.
+
+*Secure* is a discipline that works along with the [build process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.
+
+## Edge Firewall
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/overview/overview.mdx:16]:
```suggestion

---

## Edge Firewall
```

File: src/content/docs/en/pages/secure-journey/overview/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+namespace: docs_secure_overview
+menu_namespace: secureMenu
+permalink: /documentation/products/secure/overview/
+---
+
+import Button from '~/components/Button.astro'
+
+Azion Secure products assist you in protecting your domains, applications, and infrastructure.
+
+*Secure* is a discipline that works along with the [build process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/overview/overview.mdx:14]: discipline pareceu esquisito na primeira leitura, mas não tenho certeza de qual outro termo poderia ir

Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/overview/overview.mdx:14]:
```suggestion
*Secure* is a discipline that works along with the [building process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/overview/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+<Button href="/en/documentation/products/edge-firewall/" variant="secondary" text="go to edge firewall documentation"></Button>
+
+<Button href="/en/documentation/products/guides/secure/configure-main-settings/" variant="secondary" text="go to edge firewall main settings guide"></Button>
+ 
+When a user attempts to access and attack your applications hosted on Azion with any type of attack, Edge Firewall and its modules act as a protection layer before the attack reaches your application and, consequently, the origin.
+
+Edge Firewall is the core to protect your applications and domains but security rules and policies can be enhanced with [modules](/en/documentation/products/guides/secure/configure-main-settings/#modules): **DDoS Protection Unmetered**, **Edge Functions**, **Network Layer Protection**, and **Web Application Firewall**. As each of them has specific secure purposes, it's recommended to use all of them so your applications are protected with the entire stack.
+
+An attack attempt will pass through each of Edge Firewall's module and they'll block the attack according to their purposes and configured rules.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/overview/overview.mdx:30]:
```suggestion
An attack attempt will pass through each of Edge Firewall's modules and they'll block the attack according to their purposes and configured rules.
```

File: src/content/docs/en/pages/secure-journey/overview/overview.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Intelligent DNS
+
+**Intelligent DNS** allows you to host your domains on Azion's distributed infrastructure, providing high performance, availability, and security.
+
+<Button href="/en/documentation/products/intelligent-dns/" variant="secondary" text="go to edge firewall documentation"></Button>
+
+<Button href="" variant="secondary" text="go to intelligent dns main settings guide"></Button>
+
+By adding and configuring [records](/en/documentation/products/guides/secure/add-records/), your domain's information are stored securely at Azion's infrastructure, and allow you to configure other products to properly handle requests, such as edge applications and edge firewalls.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/overview/overview.mdx:48]:
```suggestion
By adding and configuring [records](/en/documentation/products/guides/secure/add-records/), your domain's information is stored securely at Azion's infrastructure and allows you to configure other products to properly handle requests, such as edge applications and edge firewalls.
```

**Review by gabriel-azion — APPROVED** [review]
Some typos and suggestions. Other than that LGTM

**Pull request merged**